### PR TITLE
Added docs and automatic deployment for docs

### PR DESCRIPTION
**Pull request by dwreeves**

### CODE REVIEW

**Comment by dwreeves** [comment]
@DMalone87 In order for the docs to work, after merging the PR, you need to go to "Settings" and change the doc source to like this:

image

Example here of what the docs look like: https://dwreeves.github.io/police-data-trust/

**Comment by dwreeves** [comment]
Also added a little helper function for dev purposes.

File: docs/src/setup.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    
+    ```
+     * Serving Flask app "backend:create_app" (lazy loading)
+     * Environment: development
+     * Debug mode: on
+     * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+     * Restarting with stat
+    ```
+
+4. Point your browser of choice to the URL shown in the terminal output. If everything worked out, the website should be running on your local computer!
```
Comment (ryanmahan) [inline docs/src/setup.md:155]:
```suggestion
3. Point your browser of choice to the URL shown in the terminal output. If everything worked out, the website should be running on your local computer!
```

Comment (dwreeves) [reply docs/src/setup.md]: Thank you!

**Review by DMalone87 — APPROVED** [review]

**Comment by DMalone87** [comment]
> @DMalone87 In order for the docs to work, after merging the PR, you need to go to "Settings" and change the doc source to like this:
> 
> image
> 
> Example here of what the docs look like: https://dwreeves.github.io/police-data-trust/

Thanks for this, @dwreeves !

**Pull request merged**